test/signs_ui/alerts/state_test.exs

```diff
@@ -47,23 +50,163 @@ defmodule SignsUi.Alerts.StateTest do
     end
   end
 
-  # to be removed, once the real implementation is in:
-  describe "handle_info(:twiddle_state)" do
-    test "rolls randomly both code paths, and works without crashing" do
-      state =
-        Enum.reduce(0..20, %SignsUi.Alerts.State{}, fn _, state ->
-          {:noreply, new_state} = SignsUi.Alerts.State.handle_info(:twiddle_state, state)
-          new_state
-        end)
+  describe "handle_events" do
```

Comment:
I think this gets us to 100% code coverage for now, but with error handling we'll need a few more failure cases. I think it would be good to have a `display_test.exs` and an `alerts_test.exs`. We generally have a 1-to-1 mapping of code modules to test modules.